src/MongoDbCache/MongoContext.cs

```diff
@@ -83,19 +82,18 @@ private async Task<CacheItem> UpdateExpiresAtIfRequiredAsync(DateTimeOffset utcN
 
             return cacheItem.WithExpiresAt(absoluteExpiration);
         }
-        #endregion
 
         public MongoContext(string connectionString, string databaseName, string collectionName)
         {
             var client = new MongoClient(connectionString);
             var database = client.GetDatabase(databaseName);
+
+            IndexKeysDefinition<CacheItem> expireAtIndexModel =
+                new IndexKeysDefinitionBuilder<CacheItem>().Ascending(p => p.ExpiresAt);
+
             _collection = database.GetCollection<CacheItem>(collectionName);
-            _collection.Indexes.CreateOne(
-                Builders<CacheItem>.IndexKeys.Ascending(x => x.ExpiresAt),
-                new CreateIndexOptions
-                {
-                    Background = true
-                });
+
+            _collection.Indexes.CreateOne(new CreateIndexModel<CacheItem>(expireAtIndexModel));
```

Comment:
I'd not remove the background index creation.
If the index gets accidentally dropped, recreating it may take a lot of time and freeze everything until done.